server/src/test/java/org/elasticsearch/cluster/ClusterStateTests.java

```diff
@@ -279,6 +280,10 @@ public void testToXContent() throws IOException {
                     "system": false,
                     "timestamp_range": {
                       "shards": []
+                    },
+                    "write_load": {
+                      "shards_write_load": [-1.0],
+                      "shards_uptime_in_millis": [-1]
```

Comment:
can we condense the names here? Might not matter too much but we do generate this xcontent frequently. Something like "loads" and "uptimes"?